### PR TITLE
feat: add dev container as option for running jupyter notebooks

### PR DESCRIPTION
**Pull request by korikuzma**

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (jsstevenson) [inline .devcontainer/devcontainer.json:1]: I know it doesn't matter but the 8-space indentation is sending me

File: notebooks/getting_started/README.md [hunk 2]
```diff
@@ -1,44 +1,141 @@
...
+`Python Environments...`, and select the first option:
+`3.12 (Python 3.12.z) venv/3.12/bin/python`
+
+_Note: Patch version may vary._
+
+### Dev Containers
+
+#### Prerequisites
+
+For your convenience, this section is copied from the
```
[the remark this answers is not in the document]
Comment (korikuzma) [reply notebooks/getting_started/README.md:51]: Ah okay. Just didn't want people to have to open a new tab

[the remark this answers is not in the document]
Comment (korikuzma) [reply notebooks/getting_started/README.md:51]: although prereqs likely isn't going to change, so I'm just going to leave if that's ok

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (korikuzma) [reply .devcontainer/devcontainer.json:1]: Finally

**Review by jsstevenson — APPROVED** [review]

**Pull request merged**